### PR TITLE
[gardening] Avoid labeled if `break` in `Repository.fromIdentifier`.

### PR DESCRIPTION
**Pull request by jdhealy**
No functional changes.

### CODE REVIEW

File: Source/CarthageKit/GitHub.swift [hunk 1]
```diff
@@ -53,14 +53,13 @@ extension Repository {
...
 		// GitHub Enterprise
-		breakpoint: if let url = URL(string: identifier), let host = url.host {
-			var pathComponents = url.pathComponents.filter { $0 != "/" }
-			guard pathComponents.count >= 2 else { break breakpoint }
-
-			// Consider that the instance might be in subdirectories.
-			let name = pathComponents.removeLast()
-			let owner = pathComponents.removeLast()
-
+		if
```
Comment (andersio) [inline Source/CarthageKit/GitHub.swift]: `guard` seems a better fit in this case. 🤔

File: Source/CarthageKit/GitHub.swift [hunk 2]
```diff
@@ -53,14 +53,13 @@ extension Repository {
...
+			let url = URL(string: identifier),
+			let host = url.host,
+			case var pathComponents = url.pathComponents.filter({ $0 != "/" }),
```
Comment (mdiep) [inline Source/CarthageKit/GitHub.swift]: *mind blown*

File: Source/CarthageKit/GitHub.swift (hunk 1 above)
Comment (jdhealy) [inline Source/CarthageKit/GitHub.swift]: @andersio done in https://github.com/Carthage/Carthage/pull/2061/commits/2426c491b73e54b5e7dc7e40312986b47e165ec2

**Comment by mdiep** [comment]
🤘

**Pull request merged**